src/components/graph/Graph.jsx

```diff
@@ -404,10 +404,19 @@ export default class Graph extends React.Component {
             width: this.state.config.width
         };
 
+        const { focusedNodeId } = this.props.data;
+
+        const containerProps = {
```

Comment:
I think it only makes sense to set `containerProps` if `focusedNodeId` is actually defined. What do you think?

Reply:
I thought about that as well, but the css `transitionDuration` property must be applied in both cases. If we don't, no animation will be displayed during zoom out when we lose focus on a node. That's the reason I'm setting `containerProps` whether there's a focused node or not, because part of its contents must always be set.

We could set the `transitionDuration` in a separate `containerStyles` variable or do something like this instead, adding the `transform` property only when required:

```
const containerProps = {
   style: {
       transitionDuration: `${this.state.config.focusAnimationLength}s`
   }
};

if (focusedNodeId) {
    containerProps.transform = graphHelper.getCenterAndZoomTransformation(focusedNodeId, this.state.d3Nodes, this.state.config);
}
```

What do you think? Do you have any other ideas? 🙂

Reply:
If that is the case let's go with the current approach